src/models/propulsion/FGTurbine.h

```diff
@@ -49,7 +52,7 @@ FORWARD DECLARATIONS
 namespace JSBSim {
 
 class Element;
-class FGFunction;
+class FGTSFC;
```

Comment:
Once the headers, will be removed you'll need to restore the declaration of `class FGFunction;`
```suggestion
class FGFunction;
```